editor/editor_node.cpp

```diff
@@ -2636,6 +2636,11 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 			quick_open->popup_dialog("Script", true);
 			quick_open->set_title(TTR("Quick Open Script..."));
 
+		} break;
+		case FILE_QUICK_OPEN_PALETTE: {
+			quick_open->popup_dialog("ColorPalette", true);
+			quick_open->set_title(TTR("Quick Open Color Palette..."));
```

Comment:
This should be in `_palette_quick_open_dialog()`. There is no reason to add `FILE_QUICK_OPEN_PALETTE` if it's not a real menu option.